docs/topics/api/auth.rst

```diff
@@ -17,9 +17,15 @@ Access Credentials
 
 To create JWTs, first obtain a **key** and **secret** from the
 `API Credentials Management Page`_.
-Keep the secret string well protected and never check it into version control.
```

Comment:
ha! I honestly didn't even notice this when I filed the bug 👀 

It looks much better as a note, thanks.